src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -2619,7 +2619,8 @@ void AuraEffect::HandleAuraUntrackable(AuraApplication const* aurApp, uint8 mode
 
 void AuraEffect::HandleAuraModPetTalentsPoints(AuraApplication const* aurApp, uint8 mode, bool /*apply*/) const
```

Comment:
The contents of the function are commented out so all of the arguments are unused. That's what's throwing the error.